packages/react/src/components/Authenticator/SelectMfa/SelectMfa.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { authenticatorTextUtil, MfaType } from '@aws-amplify/ui';
+
+import { Flex } from '../../../primitives/Flex';
+import { Heading } from '../../../primitives/Heading';
+import { Radio } from '../../../primitives/Radio';
+import { RadioGroupField } from '../../../primitives/RadioGroupField';
+import { useAuthenticator } from '@aws-amplify/ui-react-core';
+import { useCustomComponents } from '../hooks/useCustomComponents';
+import { useFormHandlers } from '../hooks/useFormHandlers';
+// import { FormFields } from '../shared/FormFields';
+import { ConfirmSignInFooter } from '../shared/ConfirmSignInFooter';
+import { RemoteErrorMessage } from '../shared/RemoteErrorMessage';
+import { RouteContainer, RouteProps } from '../RouteContainer';
+import { toLower } from 'lodash';
+
+const { getChallengeText } = authenticatorTextUtil;
+
+function parseMfaTypes(types?: MfaType[]): string {
+  if(types) {
+    return types.join();
+  } else {
+    return 'types was undefined.';
+  }
+}
+
+/*
+interface MfaOptions {
+  email: string;
+  username: string;
+}
+
+const opt: MfaOptions = {
+  email: 'EMAIL',
+  username: 'Totp Prop',
+};
+
+type MfaType = 'EMAIL' | 'wow';
+*/
+
+const generateRadioGroup = (
```

Comment:
Should we have one of the radio options selected by default?

Reply:
Or make sure the continue button is disabled until one is selected.